components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/triggerapi.go

```diff
@@ -183,3 +183,144 @@ func interfaceToIOReader(body interface{}) (io.Reader, error) {
 	}
 	return reader, nil
 }
+
+func IfHardwareEmpty(config models.Config) bool {
+	if config.Hardware.AutomateNodeCount == 0 && len(config.Hardware.AutomateNodeIps) == 0 &&
+		config.Hardware.ChefInfraServerNodeCount == 0 && len(config.Hardware.ChefInfraServerNodeIps) == 0 &&
+		config.Hardware.PostgresqlNodeCount == 0 && len(config.Hardware.PostgresqlNodeIps) == 0 &&
+		config.Hardware.OpenSearchNodeCount == 0 && len(config.Hardware.OpenSearchNodeIps) == 0 {
+		return true
+	}
+	return false
+}
+
+func NilResp(checkType string, includeOPENSEARCH bool, includePOSTGRESQL bool, includeBASTION bool) []models.CheckTriggerResponse {
```

Comment:
rename the variables, this is not we name them. It should be isOpensearch,isPostgresql, isBastion.